.github/CODEOWNERS

```diff
@@ -24,4 +25,5 @@
 /e2e_tests/tests/experiment       @determined-ai/ml-sys
 /e2e_tests/tests/nightly          @determined-ai/ml-sys
 
-/webui @determined-ai/web
```

Comment:
I kinda like the main directory and then exception near each other, similar to how /harness is done above.

Reply:
ok. fix is pushed